source/help/messaging/attaching-files.rst

```diff
@@ -51,6 +51,4 @@ Other document previews (such as Word, Excel, or PPT) are not yet supported.
 Attachment Limits and Sizes
 ---------------------------
 
-The ability to attach a maximum of 10 files per post is available today for Cloud deployments and will be available for self-hosted deployments in Mattermost Server v5.32 and later.
-
-The default maximum file size is 50 MB, but this can be changed by the System Admin.
+A maximum of 10 files can be attached per post. The default maximum file size is 50 MB, but this can be changed by the System Admin.
```

Comment:
The default is 100mb: https://github.com/mattermost/mattermost-server/pull/16668

On cloud as on Jan 26, on prem Feb 16th (v5.32)

Reply:
@esethna - The change from 50MB to 100MB is covered via https://mattermost.atlassian.net/browse/MM-32597

Reply:
@cwarnermm but it seems there will be conflicts then, since master has "100mb" and v5.32 branch has "50mb"? But I'll leave it in your hands, thanks!

Reply:
I think it should be specified that 100mb is available in Cloud on Feb 10 and later for self-managed in v5.33.